internal/report/report.go

```diff
@@ -89,6 +91,10 @@ func (r *Reporter) SendReport(ctx context.Context) (*DataResponse, error) {
 	rCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
 	defer cancel()
 
+	var m runtime.MemStats
+	runtime.ReadMemStats(&m)
+	r.body["heap_usage_bytes"] = strconv.FormatUint(m.Alloc, 10)
```

Comment:
❓ Why only `Alloc` (the same as `HeapAlloc`), and not `HeapInUse` or `TotalAlloc` or `Sys`? 💭

Reply:
Heap usage is a good indicator of OPA's memory consumption. If you think any other memory stats are useful, happy to add them as well.

Reply:
I'm not sure! Not my strong suit. 

```
	// HeapSys is bytes of heap memory obtained from the OS.
	//
	// HeapSys measures the amount of virtual address space
	// reserved for the heap. This includes virtual address space
	// that has been reserved but not yet used, which consumes no
	// physical memory, but tends to be small, as well as virtual
	// address space for which the physical memory has been
	// returned to the OS after it became unused (see HeapReleased
	// for a measure of the latter).
	//
	// HeapSys estimates the largest size the heap has had.
	HeapSys [uint64](https://pkg.go.dev/builtin#uint64)
```

_"HeapSys estimates the largest size the heap has had."_ sounds interesting. 🤔